convert2rhel/actions/pre_ponr_changes/kernel_modules.py

```diff
@@ -67,8 +76,26 @@ def _get_rhel_supported_kmods(self):
         if system_info.version.major == 8:
             basecmd.append("--setopt=module_platform_id=platform:el8")
 
+        active_repos = ["--disablerepo=*"]
```

Comment:
This is tricky. What if the user uses the `--disablerepo` c2r option?

We handle it in the `call_yum_cmd` (https://github.com/oamg/convert2rhel/blob/v1.3.2/convert2rhel/pkghandler.py#L123).

Reply:
I think `--disablerepo` from the c2r options is safe to ignore as `--disablerepo=*` is a superset of anything that the user might have specified.

OTOH, prior to this change, we didn't disable any repos when running repoqery in this function.  I'm not sure how that worked... It seems like we would be running repoquery using both the vendor's repos and the RHEL repos in that case, but we only want to find kernel modules that are in packages in the RHEL repos here?

Reply:
> I think --disablerepo from the c2r options is safe to ignore as --disablerepo=* is a superset of anything that the user might have specified.

Ah, right, that makes sense.

> It seems like we would be running repoquery using both the vendor's repos and the RHEL repos in that case, but we only want to find kernel modules that are in packages in the RHEL repos here?

Yes, we want to query just the RHEL repos. Previously we've been using the `--repoid` option.

```
--repoid=<repo>
     Specify  which  repository  to query. Using this option disables all repositories not explicitly enabled with --repoid option (can be used multiple times). By default repoquery uses whatever repositories are enabled in YUM configuration.
```
So we were really querying just the RHEL repos. Now given that yum doesn't have this option and you've switched to using `--enablerepo` then passing `--disablerepo=*` makes sense.